new/src/index.tsx

```diff
@@ -1,29 +1,16 @@
-import React from "react";
-import ReactDOM from "react-dom/client";
-import "./index.css";
-import App from "./App";
-import reportWebVitals from "./reportWebVitals";
-import i18n from "i18next";
-import { initReactI18next } from "react-i18next";
-import en from "./i18n/en.json";
-
-const root = ReactDOM.createRoot(document.getElementById("root") as HTMLElement);
-
-i18n.use(initReactI18next).init({
```

Comment:
why did you dump the i18n?

Reply:
See above haha. Let's just add it again.